python/hail/dataset.py

```diff
@@ -2940,6 +2940,84 @@ def query_variants(self, exprs):
         r, t = self.query_variants_typed(exprs)
         return r
 
+    def query_genotypes_typed(self, exprs):
+        """Perform aggregation queries over genotypes, and returns python objects and types.
+
+        **Examples**
+
+        >>> gq_hist, t = vds.query_genotypes_typed(
+        ...     'gs.map(g => g.gq).hist(0, 100, 100)')
+
+        See :py:meth:`.query_genotypes` for more information.
```

Comment:
This is how we've done the others.  I'll push this in in the same fashion and we can change the others at another time.